packages/atomic/src/components/atomic-breadbox/atomic-breadbox.tsx

```diff
@@ -51,6 +55,8 @@ export class AtomicBreadbox implements InitializableComponent {
   private resizeObserver?: ResizeObserver;
   private showMore!: HTMLButtonElement;
   private showLess!: HTMLButtonElement;
+  private lastRemovedBreadcrumbIndex = 0;
+  private numberOfBreadcrumbs = 0;
```

Comment:
Neither of these values needs an `undefined` state nor needs to be reset, since they are only used after `focusAfterSearch` was called.